### ISSUE TITLE
Work around file manager apps not recognizing pkpass

### ISSUE THREAD

**Issue by TheLastProject** [CatimaLoyalty/Android#2220]
Some file managers seem to not correctly use `application/vnd.apple.pkpass` as mime type for pkpass files, making Catima not show up as option.

We should try to add a workaround for users on affected file managers.

**Comment by VasilisKos** [comment]
Confirmed on Solid Explorer.

**Comment by TheLastProject** [comment]
Interesting note: for email this seems to often be the *sending* client that decides the format, not the receiving client: https://github.com/thunderbird/thunderbird-android/issues/1357#issuecomment-217667602

**Comment by erAck** [comment]
Yes, the sending mail agent decides on Content-Type. In the case of application/octet-stream the receiving agent might apply some intelligence magic..
